### ISSUE TITLE
[Feature request]: debug support for vim? 

### ISSUE THREAD

**Issue by ubaldot** [platformio/platformio-core#4825]
I just discovered that it is possible to debug with platoformio and I love such a 1-click concept! :smiley:
However, I have seen that it is possible with VSCode and I am wondering if it is also possible with Vim. AFAIK, there is [vimspector](https://github.com/puremourning/vimspector) which seems to act as a Debug Adapter Protocol (DAP) client, or the builtin `:termdebug.`

**Comment by ivankravets** [comment]
Debugging should work. I previously have used Vim/Emacs for so many years and tested PlatformIO Debugging. The CLI command is `pio debug --interface gdb -x .pioinit`

**Comment by ubaldot** [comment]
Ok, would that be possible to add a guideline in the docs? I could try in the meantime 🙂

**Comment by ivankravets** [comment]
Yes, we would be thankful for the PR. The raw page is hosted here https://github.com/platformio/platformio-docs/blob/develop/integration/ide/vim.rst
